apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/(analysis)/summary/components/shareEmbedTabs/EmailTab.tsx

```diff
@@ -215,12 +219,15 @@ const getEmailTemplate = (survey: TSurvey, surveyUrl: string, brandColor: string
               </Section>
             </Container>
             {/* {!firstQuestion.required && (
-                <EmailButton
-                  href={`${urlWithPrefilling}${firstQuestion.id}=dismissed`}
-                  className="mt-4 cursor-pointer appearance-none rounded-md bg-brand-color px-6 py-3 text-sm font-medium text-white">
-                  {firstQuestion.buttonLabel || "Skip"}
-                </EmailButton>
-              )} */}
+              <EmailButton
+                href={`${urlWithPrefilling}${firstQuestion.id}=dismissed`}
+                className={cn(
+                  "bg-brand-color mt-4 cursor-pointer appearance-none rounded-md px-6 py-3 text-sm font-medium",
+                  isLight(brandColor) ? "text-black" : "text-white"
+                )}>
+                {firstQuestion.buttonLabel || "Skip"}
+              </EmailButton>
+            )} */}
```

Comment:
Can we remove this piece out if its not needed?

Reply:
Hi @ShubhamPalriwala ! I've added this code to support prefilling for optional questions. There is one edge case in prefiling for optional questions, will fix that and uncomment this soon.

Reply:
Can you elaborate on the edge case? And whats your plan on making this handle that as well?

Reply:
For any non-required question, we can press next to navigate to the next question, and this doesn't cause any entry in the response(related to skipped ques), but in case of prefiling, we are not handling the behavior to proceed to the next question.
I am planning to handle this as if we receive the prefiling value as "skipped" for any not-required ques, then will navigate to the next question without running the prefiling logic further.
Hence, hide the skip buttons from email templates.